app/assets/stylesheets/modules/sitter.scss

```diff
@@ -163,17 +163,39 @@
     }
   }
 
-   .price {
-      .small-9{
-        @include sitter-details;
-        font-size: 0.9em;
-      }
+ .price {
+    .small-9{
```

Comment:
Selector should have depth of applicability no greater than 2, but was 3<br>Opening curly brace `{` should be preceded by one space